### PR TITLE
chore: rename index.ts for systemjs

### PR DESCRIPTION
**Pull request by Foxandxss**
Having `index.js` per component forces the end user to have an entry at `systemjs config`. So if we have 20 components, we force the users to have 20 entries.

I chose `foo.module.ts` as the name.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
-import {NgbPopoverModule} from './popover';
-import {NgbProgressbarModule} from './progressbar';
-import {NgbRatingModule} from './rating';
-import {NgbTabsetModule} from './tabset';
-import {NgbTimepickerModule} from './timepicker';
-import {NgbTooltipModule} from './tooltip';
-import {NgbTypeaheadModule} from './typeahead';
-
-export {NgbPanelChangeEvent} from './accordion';
-export {NgbTabChangeEvent} from './tabset';
```
Comment (pkozlowski-opensource) [inline src/index.ts]: Those exports are still needed. Did you remove them by accident?

Comment (Foxandxss) [inline src/index.ts]: No, just added them with the other imports, line 3 and 13.

Comment (pkozlowski-opensource) [inline src/index.ts]: Huh? Those are _exports_....

Comment (Foxandxss) [inline src/index.ts]: Ah, completely blind.

**Comment by pkozlowski-opensource** [comment]
I've changed the remaining parts and merged.

**Pull request closed without merging**